applications/MeshingApplication/python_registry_lists.py

```diff
@@ -0,0 +1,11 @@
+python_modelers_to_be_registered = [
+    "modelers.convert_linear_tetrahedra_to_quadratic_modeler.ConvertLinearTetrahedraToQuadraticModeler",
```

Comment:
```suggestion
    "convert_linear_tetrahedra_to_quadratic_modeler.ConvertLinearTetrahedraToQuadraticModeler",
```
The `modelers` keyword is automatically deduced from the list. As it is right now we'd be registering `modelers.modelers`. Aside of this, the rest is correct.

Reply:
If I understood the logic correctly, this is the path to the python file (it is in the python_scripts/modelers/ subfolder). As far as I can tell, this is the same approach used for the modelers in the kratos core. The class is currently being registered with the `"Modelers.KratosMultiphysics.MeshingApplication.ConvertLinearTetrahedraToQuadraticModeler"` key (I am using it in the tests), which I believe is the standard approach (?)

Reply:
Sure. I didn't realize about the subfolder. You're absolutely right. Forget about my comment.